esphomeyaml/guides/faq.rst

```diff
@@ -197,6 +197,48 @@ Devices that are technically already supported, but for which guides will be cre
 Anything missing? I'd be happy to chat about more integrations over on the `discord channel
 <https://discord.gg/KhAMKrd>`__ - no guarantees that everything will be supported though!
 
+I can't update using OTA because of to little space, now what?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you are using ESP8266/Sonoff devices you will probably encounter this error during OTA update:
+
+`ERROR [esphomeyaml.espota] Bad Answer: ERR: ERROR[4]: Not Enough Space`
+
+This is because of the limited amount of flash memory available on these devices (often just 1M).
```

Comment:
So it might be good to give indications what usually takes up a whole lot of memory. Especially the web server often uses up a whole lot of space.

Maybe also add a comment here or further down that disabling the web server can help (I mean the web interface isn't that great anyway and I *think* few people are actually using it)

Reply:
Disabling the webserver was my first approach as well, but then I ran into one config that even then was to big, so I found this solution :) I'll add some more extra explaining as well.

Reply:
I've also been thinking on how to make the size of each feature visible, don't know if there currently is some smartness in platformio for this, otherwise something I think might work is a script that compiles a config with one feature enabled at a time and records all the different sizes so we can put that in a table or something.